test/httpoison_test.exs

```diff
@@ -94,6 +94,17 @@ defmodule HTTPoisonTest do
     assert_response HTTPoison.get("https://localhost:8433/get", [], ssl: [cacertfile: cacert_file, keyfile: key_file, certfile: cert_file])
   end
 
+  test "http+unix scheme" do
+    if Application.get_env(:httparrot, :unix_socket, false) do
```

Comment:
So by default it's false and AFAIK we don't set this to true. How do we know this test is running?